### PR TITLE
zh-ch: upate 'HTML table advanced features and accessibility'

### PR DESCRIPTION
**Pull request by abing22333**
### Description

upate 'HTML table advanced features and accessibility'

### CODE REVIEW

**Comment by abing22333** [comment]
在 [Adding structure with \<thead>, \<tbody>, and \<tfoot>](https://developer.mozilla.org/en-US/docs/Learn/HTML/Tables/Advanced#adding_structure_with_thead_tbody_and_tfoot) 的开始，强调了要按顺序使用这三个元素，
> To use them, they should be included in the following order:
> 
> * The <thead> element must wrap the part of the table that is the header — this is usually the first row containing the column headings, but this is not necessarily always the case. If you are using [<col>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/col)/[<colgroup>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/colgroup) elements, the table header should come just below those.
> * The <tbody> element needs to wrap the main part of the table content that isn't the table header or footer.
> * The <tfoot> element needs to wrap the part of the table that is the footer — this might be a final row with items in the previous rows summed, for example.

但是接下来的 [Active learning: Adding table structure](https://developer.mozilla.org/en-US/docs/Learn/HTML/Tables/Advanced#active_learning_adding_table_structure) 中没有按顺序使用。

> 1. First of all, make a local copy of [spending-record.html](https://github.com/mdn/learning-area/blob/main/html/tables/advanced/spending-record.html) and [minimal-table.css](https://github.com/mdn/learning-area/blob/main/html/tables/advanced/minimal-table.css) in a new folder.
> 2. Try opening it in a browser — You'll see that it looks OK, but it could stand to be improved. The "SUM" row that contains a summation of the spent amounts seems to be in the wrong place, and there are some details missing from the code.
> 3. Put the obvious headers row inside a `<thead>` element, the "SUM" row inside a `<tfoot>` element, and the rest of the content inside a `<tbody>` element.

前后自相矛盾了，到时候抽空给 content 提个pr

File: files/zh-cn/learn/html/tables/advanced/index.md [hunk 1]
```diff
@@ -43,30 +41,31 @@ slug: Learn/HTML/Tables/Advanced
...
-> **备注：** [`summary`](/zh-CN/docs/Web/HTML/Element/table#summary) 属性也可以在 `<table>` 元素中使用，用来提供一段描述，同样可以被屏幕阅读设备阅读。我们推荐使用 `<caption>` 元素来代替，因为 `summary` 已经废弃了，且不能被视力正常的用户阅读（它不会出现在页面上）。
+> [!NOTE]
+> 同样，[`summary`](/zh-CN/docs/Web/HTML/Element/table#summary) 属性也可以在 `<table>` 元素中使用，用来提供一段描述，可以被屏幕阅读设备阅读。但是我们推荐使用 `<caption>` 元素来代替，因为 `summary` 已经废弃了，且不能被视力正常的用户阅读（它不会出现在页面上）。
 
 ### 动手练习：添加一个标题
 
 我们来试试看吧，回顾一下我们在之前的文章中第一次遇到的例子。.
 
-1. 打开你的语言老师的学校时间表，就是 [HTML 表格基础](/zh-CN/docs/Learn/HTML/Tables/Basics#动手练习：colgroup_and_col) 结尾中的例子，或者把 [timetable-fixed.html](https://github.com/mdn/learning-area/blob/main/html/tables/basic/timetable-fixed.html) 文件复制到本地。
+1. 打开你的语言老师的学校时间表，就是 [HTML 表格基础](/zh-CN/docs/Learn/HTML/Tables/Basics#colgroup%20和%20col) 结尾中的例子，或者把 [timetable-fixed.html](https://github.com/mdn/learning-area/blob/main/html/tables/basic/timetable-fixed.html) 文件复制到本地。
```
Comment (yin1999) [inline files/zh-cn/learn/html/tables/advanced/index.md:51]: 这里的链接是正确的，可以说一下为啥要替换它吗？

File: files/zh-cn/learn/html/tables/advanced/index.md [hunk 2]
```diff
@@ -43,30 +41,31 @@ slug: Learn/HTML/Tables/Advanced
...
 这些元素不会使表格更易于屏幕阅读器用户访问，也不会造成任何视觉上的改变。然而，它们在应用样式和布局上会起到作用，可以更好地让 CSS 应用到表格上。给你一些有趣的例子，在长表格的情况下，你可以在每个打印页面上使表格页眉和页脚重复，你也可以让表格的正文部分显示在一个单独的页面上，并通过上下滚动来获得内容。
 
-试着使用它们：
+要使用它们，应按以下顺序排列：
 
-- `<thead>` 元素必须包住表格中作为表头的部分。一般是第一行，往往都是每列的标题，但是不是每种情况都是这样的。如果你使用了 {{htmlelement("col")}}/{{htmlelement("colgroup")}} 元素，那么 `<thead>` 元素就需要放在它们的下面。
-- `<tfoot>` 元素需要包住表格中作为表脚的部分。一般是最后一行，往往是对前面所有行的总结，比如，你可以按照预想的方式将`<tfoot>`放在表格的底部，或者就放在 `<thead>` 的下面。(浏览器仍将它呈现在表格的底部)
-- `<tbody>` 元素需要包住表格内容中不在表头或表尾的其他部分。它可以出现在表头的下方，或者有时出现在表脚下方，这取决于你如何安排它。
+- `<thead>` 元素必须包住表格的表头部分。一般是第一行，往往都是每列的标题，但是不是每种情况都是这样的。如果你使用了 {{htmlelement("col")}}/{{htmlelement("colgroup")}} 元素，那么 `<thead>` 元素就需要放在它们的下面。
+- `<tbody>` 元素需要包住表格内容的主要部分，不是表头和表尾。
```
Comment (yin1999) [inline files/zh-cn/learn/html/tables/advanced/index.md:67]:
```suggestion
- `<tbody>` 元素需要包住表格内容的主要部分（不是表头和表尾）。
```

File: files/zh-cn/learn/html/tables/advanced/index.md [hunk 3]
```diff
@@ -363,49 +250,82 @@ th {
...
-  <td>Great idea</td>
+  <td>好主意</td>
   <td>30</td>
 </tr>
 ```
 
 屏幕阅读器会识别这种结构化的标记，并一次读出整列或整行。
 
-`scope` 还有两个可选的值： `colgroup` 和 `rowgroup`。这些用于位于多个列或行的顶部的标题。如果你回顾这部分文章开始部分的“2016 年 8 月出售的物品”表格，你会看到“衣物”单元格在“长裤”、“衬衫”和“裙子”单元格的上面。这几个单元格都应该被标记为表头（`<th>`），但是“衣物”是一个位于顶部且定义了其他三个子标题的表头。因此“衣物”应该有一个 `scope="colgroup"` 属性，而另外三个子标题应该有 `scope="col"` 属性。
+`scope` 还有两个可选的值： `colgroup` 和 `rowgroup`。这些用于位于多个列或行的顶部的标题。如果你回顾这部分文章开始部分的“2016 年 8 月出售的物品”表格，你会看到“衣物”单元格在“长裤”、“裙子”和“连衣裙”单元格的上面。这几个单元格都应该被标记为标题（`<th>`），但是“衣物”是一个位于顶部且定义了其他三个子标题的标题。因此“衣物”应该有一个 `scope="colgroup"` 属性，而另外三个子标题应该有 `scope="col"` 属性：
```
Comment (yin1999) [inline files/zh-cn/learn/html/tables/advanced/index.md:263]:
```suggestion
`scope` 还有两个可选的值：`colgroup` 和 `rowgroup`。这些用于位于多个列或行的顶部的标题。如果你回顾这部分文章开始部分的“2016 年 8 月出售的物品”表格，你会看到“衣物”单元格在“长裤”、“裙子”和“连衣裙”单元格的上面。这几个单元格都应该被标记为标题（`<th>`），但是“衣物”是一个位于顶部且定义了其他三个子标题的标题。因此“衣物”应该有一个 `scope="colgroup"` 属性，而另外三个子标题应该有 `scope="col"` 属性：
```

File: files/zh-cn/learn/html/tables/advanced/index.md [hunk 4]
```diff
@@ -363,49 +250,82 @@ th {
...
 
-1. 为每个 `<th>` 元素添加一个唯一的 `id` 。
-2. 为每个 `<td>` 元素添加一个 `headers` 属性。每个单元格的 `headers` 属性需要包含它从属于的所有标题的 id，之间用空格分隔开。
+`headers` 属性包含一个空格分隔的无序字符串列表，每个字符串对应 `<th>` 元素的唯一 `id`，这些 `<th>` 元素为数据单元格（`<td>` 元素）或另一个标题单元格（`<th>` 元素）提供标题。
 
-这会给你的 HTML 表格中每个单元格的位置一个明确的定义。像一个电子表格一样，通过 headers 属性来定义属于哪些行或列。为了让它工作良好，表格同时需要列和行标题。
+这样，HTML 表格就明确定义了表格中每个单元格的位置，这些单元格由其所属的每列和每行的标题定义，有点像电子表格。为使其运行良好，表格确实需要列标题和行标题。
 
-回到我们的花费成本示例，前两个片段可以重写为：
+回到“2016 年 8 月出售的物品”示例，可以使用 id 和 headers 属性：
```
Comment (yin1999) [inline files/zh-cn/learn/html/tables/advanced/index.md:304]:
```suggestion
回到“2016 年 8 月出售的物品”示例，可以这样使用 `id` 和 `headers` 属性：
```

File: files/zh-cn/learn/html/tables/advanced/index.md [hunk 5]
```diff
@@ -363,49 +250,82 @@ th {
...
+
+1. 为每个 `<th>` 元素添加一个唯一的 `id`。
+2. 为每个作为子标题（即在其上方有一个标题元素）的 `<th>` 元素添加一个 `headers` 属性。其值是位于顶部并定义子标题的标题 `id`，在我们的示例中，列标题为 "衣物"，行标题为 "比利时"。
```
Comment (yin1999) [inline files/zh-cn/learn/html/tables/advanced/index.md:307]:
```suggestion
2. 为每个作为子标题（即在其上方有一个标题元素）的 `<th>` 元素添加一个 `headers` 属性。其值是位于顶部并定义子标题的标题 `id`，在我们的示例中，列标题为“衣物”，行标题为“比利时”。
```

File: files/zh-cn/learn/html/tables/advanced/index.md [hunk 6]
```diff
@@ -363,49 +250,82 @@ th {
...
+3. 为每个 `<td>` 元素添加一个 `headers` 属性，并以空格分隔的列表形式添加相关联的 `<th>` 元素的 `id`。像在电子表格中一样进行操作：找到数据单元格，然后搜索行和列的相应标题。指定 id 的顺序并不重要，但应保持一致，使其井井有条。
```
Comment (yin1999) [inline files/zh-cn/learn/html/tables/advanced/index.md:308]:
```suggestion
3. 为每个 `<td>` 元素添加一个 `headers` 属性，并以空格分隔的列表形式添加相关联的 `<th>` 元素的 `id`。像在电子表格中一样进行操作：找到数据单元格，然后搜索行和列的相应标题。指定 `id` 的顺序并不重要，但应保持一致，使其井井有条。
```

File: files/zh-cn/learn/html/tables/advanced/index.md (hunk 1 above)
Comment (abing22333) [reply files/zh-cn/learn/html/tables/advanced/index.md:51]: `动手练习：colgroup_和_col` 才是正确的

**Review by abing22333 — COMMENTED** [review]
fix bug

File: files/zh-cn/learn/html/tables/advanced/index.md [hunk 7]
```diff
@@ -415,8 +335,8 @@ th {
 ### 动手练习：使用 scope 和 headers
 
 1. 对于这个最后的练习，首先把 [items-sold.html](https://github.com/mdn/learning-area/blob/main/html/tables/advanced/items-sold.html) 和 [minimal-table.css](https://github.com/mdn/learning-area/blob/main/html/tables/advanced/minimal-table.css) 复制到你的本地环境。
-2. 现在尝试添加适当的 `scope` 属性来让表格变得更加恰当。
-3. 最后，尝试把未添加 `scope` 属性的源文件再复制一份。这次使用 `id` 和 `headers` 属性让表格变得更加恰当。
+2. 现在尝试添加适当的 `scope` 属性来让表格变得更加易于访问。
+3. 最后，尝试把未添加 `scope` 属性的源文件再复制一份。这次使用 `id` 和 `headers` 属性创建精确、明确的关联，使表格变得更加易于访问。
 
 > [!NOTE]
 > 你可以根据我们完成的例子检查你的工作，请看 [items-sold-scope.html](https://github.com/mdn/learning-area/blob/main/html/tables/advanced/items-sold-scope.html) ([also see this live](https://mdn.github.io/learning-area/html/tables/advanced/items-sold-scope.html)) 和 [items-sold-headers.html](https://github.com/mdn/learning-area/blob/main/html/tables/advanced/items-sold-headers.html)（[查看在线运行版本](https://mdn.github.io/learning-area/html/tables/advanced/items-sold-headers.html)）。
```
Comment (yin1999) [inline files/zh-cn/learn/html/tables/advanced/index.md:342]:
```suggestion
> 你可以根据我们完成的例子检查你的工作，请看 [items-sold-scope.html](https://github.com/mdn/learning-area/blob/main/html/tables/advanced/items-sold-scope.html)（[[查看在线运行版本](https://mdn.github.io/learning-area/html/tables/advanced/items-sold-scope.html)）和 [items-sold-headers.html](https://github.com/mdn/learning-area/blob/main/html/tables/advanced/items-sold-headers.html)（[查看在线运行版本](https://mdn.github.io/learning-area/html/tables/advanced/items-sold-headers.html)）。
```

**Review by yin1999 — APPROVED** [review]
:+1:

**Pull request merged**